docs/pre-release/doc-links.yml

```diff
@@ -8,6 +8,8 @@
        link: install/upgrade/
      - title: Install Traffic Manager with Helm
        link: install/helm/
+     - title: Telepresence Pro
+       link: install/telepresence-pro/
```

Comment:
I'm not clear from this PR why we need to add Telepresence Pro?

Reply:
Oops. That change shouldn't be there.